src/dstack/_internal/server/settings.py

```diff
@@ -26,6 +26,10 @@
 ALEMBIC_MIGRATIONS_LOCATION = os.getenv(
     "DSTACK_ALEMBIC_MIGRATIONS_LOCATION", "dstack._internal.server:migrations"
 )
+# Users may want to increase pool size to support more concurrent resources
+# if their db supports many connections
+DB_POOL_SIZE = int(os.getenv("DSTACK_DB_POOL_SIZE", 10))
+DB_MAX_OVERFLOW = int(os.getenv("DSTACK_DB_MAX_OVERFLOW", 10))
```

Comment:
Not sure if these vars should be documented, but there is a dedicated section “Internal environment variables” https://dstack.ai/docs/reference/environment-variables/#server